viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+
+
+
+from webbrowser import get
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie = None
+    if title == None or genre == None or rating == None:
+        movie == None
+    else:
+        movie = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+        }
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if title in movie["title"]:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+    return user_data
 
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
 
+def get_watched_avg_rating(user_data):
+    sum = 0
+    counter = 0
+    average = 0.0
+    for movie in user_data["watched"]:
+        if movie["rating"] == False:
+            average == 0.0
+        else:
+            sum += movie["rating"]
+            counter += 1
+            average = sum / counter

```

Comment:
Creating a `counter` variable works but you can also use `len(user_data["watched"])`